docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -380,9 +380,8 @@ paths:
             PDF fingerprint.
 
             `*` will match any character sequence (including an empty one), 
-            and a `_` will match any single character. `*`s are only permitted 
-            within the path and query parts of the URI and `_`s are only permitted
-            within the domain, path, and query parts of the URI.
+            and a `_` will match any single character. Wildcards are only permitted 
```

Comment:
I was editing some API docs today and noticed that your editor appears to be adding/allowing trailing whitespace on the ends of some lines (like this one). Possible to change your editor's config to disallow this to match conventions?

Reply:
Ah I see, I need to add this file type to the list of files I remove trailing whitespace from. Thanks for pointing this out!